app/serializers/provider_serializer.rb

```diff
@@ -95,13 +95,17 @@ def select_value_for_provider(provider_code, values)
 
   def generate_provider_contacts
     provider_contacts = object.contacts.map do |c|
-      c.attributes.slice('type', 'name', 'email', 'telephone')
+      contact = c.attributes.slice('type', 'name', 'email', 'telephone')
+      if c.type == 'admin'
```

Comment:
So is the data store as admin or admin_contact as the if looks like a workaround  code.

Reply:
This is for all intents and purposes workaround code, the type field is stored as 'admin' and translated to 'admin_contact' on serialisation.

Reply:
> So is the data store as admin or admin_contact as the if looks like a workaround code.

Ya, we decided to call it `admin` in our "clean" data model, but then forgot to document it that way. So now we're just adjusting it for the v1 api for UCAS.

Reply:
fine enough, workaround code it be